modules/webbrowser/include/modules/webbrowser/webbrowserclient.h

```diff
@@ -211,7 +220,7 @@ class IVW_MODULE_WEBBROWSER_API WebBrowserClient : public CefClient,
 
 protected:
     struct BrowserData {
-        Processor* processor = nullptr;
+        Processor* processor;
```

Comment:
Is this only use in the main thread or how is access synchronized?

Reply:
Yes, should only be on the main thread